### ISSUE TITLE
some small problems about the codes~

### ISSUE THREAD

**Issue by huobaoxiaopipi** [BUPT-GAMMA/PTHGNN#3]
Hi, thanks for providing the awesome code of PT-HGNN.
After reading the paper, I'm trying to run your code on my own graph dataset. But I have some small problems:
1. Compared with the paper, I am a little confused about the difference betweeen pretrain.py and pretrain_structure.py. Could you add some detailed guidelines in readme?
2. The actual implementation of the two parts of loss function in the code seems to be different in the paper. loss_link and loss_attr seem to be the loss functions designed in the paper of GPT-GNN?
